src/forms/formHelpers.js

```diff
@@ -377,7 +386,7 @@ class ManagedNumberField extends Component {
         valid = this.props.validate( value );
 
     if ( valid ) {
-      this.props.store( evnt, value, this.props.interval );
+      this.props.store( evnt, inputProps, this.props.otherData );
```

Comment:
Not sure why you're passing `inputProps` here instead of `value`. But then again I don't understand why Semantic UI includes these in the first place, much less merged with the value (which is already available on the event). So there's probably something I'm missing here.

Reply:
`evnt` isn't a representation of an input event. It's sort of like a Redux action. (One of the other files has where this is used - will update the comment when I find it.)

Reply:
I think the `inputProps` are the props that you give to the React component, not just what's on the element. I could be wrong. I saw no reason not to pass along the whole thing. Is this a blocking issue?

Reply:
No, its not a big deal.

`inputProps` are the props of `<Input>` merged with `evnt.target.value`. [source](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/elements/Input/Input.js#L135-L138)

Reply:
Thanks for the further info! Looks like it at least includes the Component props.